client/web-sveltekit/src/routes/[...repo=reporev]/(validrev)/(code)/+layout.svelte

```diff
@@ -212,16 +212,28 @@
                     </button>
                 </Tooltip>
 
-                <RepositoryRevPicker
-                    display="block"
-                    repoURL={data.repoURL}
-                    revision={data.revision}
-                    commitID={data.resolvedRevision.commitID}
-                    defaultBranch={data.defaultBranch}
-                    getRepositoryBranches={data.getRepoBranches}
-                    getRepositoryCommits={data.getRepoCommits}
-                    getRepositoryTags={data.getRepoTags}
-                />
+                {#if data.isPerforceDepot}
+                    <RepositoryRevPicker
+                        display="block"
+                        repoURL={data.repoURL}
+                        revision={data.revision}
+                        commitID={data.resolvedRevision.commitID}
+                        defaultBranch={data.defaultBranch}
+                        latestChangelistID={data.latestChangelistID}
+                        getDepotChangelists={data.getDepotChangelists}
+                    />
```

Comment:
If the corresponding changes this could be
```suggestion
                    <RepositoryRevPicker
                        display="block"
                        repoURL={data.repoURL}
                        commitID={data.latestChangelistID}
                        getDepotChangelists={data.getDepotChangelists}
                    />
```

Reply:
I'll work on this next. I'd like to merge as is for now.